+bisos/+iteration/+step/termination.m

```diff
@@ -0,0 +1,111 @@
+classdef termination < bisos.iteration.Step
+   
+properties
+    type = 'termination';
+    varout = [];
+    varin;
+    data_prev = true;
+    
+    % save function handles and variables in play (maybe a struct would be nice)
+    operator;
+    
+    fhan1; 
+    vars1; % previous variables
+    
+    fhan2;
+    vars2; % current variables
+    
+    
+    
+end
+
+methods
+    function step = termination(~, previous, current, op, varargin)
+        % setup termination rule
+        
+        % get variabled needed as input in a certain cycle and save
+        % function handles and other variables needed
+        if length(current)==1
+            % in case NO function handle is defined
+            for i = 1:length(current)
+                step.varin{length(step.varin)+1}=current{i};
+            end
+            step.vars2 = current;
+        else
+            % in case some function handle is defined
+            for i = 1:length(current{2})
+                step.varin{length(step.varin)+1}=current{2}{i};
+            end
+            step.vars2 = current{2};
```

Comment:
do you only allow for a single argument to `fhan`?

Reply:
I revisited the code and currently the part related to the `fhan` isn't implemented yet completely, if you check this step `run` no `fhan` is considered. 
The total implementation of the `fhan` is postponed as stated in the Readme